downstream/modules/builder/con-about-ee.adoc

```diff
@@ -0,0 +1,21 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="con-about-ee"]
+
+= About {ExecEnvName}
+
+[role="_abstract"]
+
+{ExecEnvNameStart} are container images on which all automation in {PlatformName} is run. {ExecEnvName} create a common language for communicating automation dependencies, and provide a standard way to build and distribute the automation environment.
```

Comment:
<img width="964" alt="Screenshot 2022-08-30 at 12 59 15" src="https://user-images.githubusercontent.com/633765/187376540-e11aabd7-007f-4cd5-8ef0-d6db88b7be99.png">

{ExecEnvName} should start with a capital letter.

Reply:
Change complete.